setup/setup.go

```diff
@@ -72,6 +77,10 @@ func (o *Options) BindFlags(fs *flag.FlagSet) {
 	fs.IntVar(&o.MaxConcurrentReconciles, "max-concurrent-reconciles", 1, "Defines the maximum number of concurrent reconciles for all controllers.")
 	fs.BoolVar(&o.CleanUpOldLogFile, "cleanup-old-cli-logs", true, "Defines if the operator should delete old fdbcli log files.")
 	fs.DurationVar(&o.LogFileMinAge, "log-file-min-age", 5*time.Minute, "Defines the minimum age of fdbcli log files before removing when \"--cleanup-old-cli-logs\" is set.")
+	fs.IntVar(&o.LogFileMaxAge, "log-file-max-age", 28, "Defines the maximum age to retain old operator log file in number of days.")
+	fs.IntVar(&o.LogFileMaxSize, "log-file-max-size", 250, "Defines the maximum size in megabytes of the operator log file before it gets rotated.")
+	fs.IntVar(&o.MaxNumberOfOldLogFiles, "max-old-log-files", 3, "Defines the maximum number of old operator log files to retain.")
+	fs.BoolVar(&o.CompressOldFiles, "compress", false, "It determines if the rotated log files should be compressed using gzip or not.")
```

Comment:
```suggestion
	fs.BoolVar(&o.CompressOldFiles, "compress", false, "Defines if the rotated log files should be compressed using gzip or not.")
```

Just for consistency

Reply:
Yes I tested the changes locally